corehq/apps/es/tests/utils.py

```diff
@@ -76,3 +79,18 @@ def populate_es_index(models, index_name, doc_prep_fn=lambda doc: doc):
 
 def populate_user_index(users):
     populate_es_index(users, 'users')
+
+
+def case_search_es_setup(domain, case_blocks):
+    """Submits caseblocks, creating the cases, then sends them to ES"""
+    from corehq.apps.hqcase.utils import submit_case_blocks
+    _, cases = submit_case_blocks([cb.as_text() for cb in case_blocks], domain=domain)
+    order = {cb.case_id: index for index, cb in enumerate(case_blocks)}
+    # send cases to ES in the same order they were passed in
```

Comment:
Why is this important? I see in the other comment it says it's to get the `indexed_on` field to match the creation order by why do we care about that?

Reply:
It was so the results would be returned in a predictable order for tests (`submit_case_blocks` doesn't preserve order).  This is used in the test suite for the case list API, which paginates by `indexed_on` - this way I know which cases fall on which side of each page break.  Do you think that's leaking implementation across the abstraction boundary?

Reply:
I think it's OK but maybe just update the comment to say 'why'